pkg/util/admission/admissionpb/io_threshold.go

```diff
@@ -42,9 +41,26 @@ func (iot *IOThreshold) Score() (float64, bool) {
 	if iot == nil || iot.L0NumFilesThreshold == 0 || iot.L0NumSubLevelsThreshold == 0 {
 		return 0, false
 	}
+	numSubLevels := iot.L0NumSubLevels
+	if iot.L0MinimumSizePerSubLevel > 0 {
+		// Upper-bound on number of sub-levels. See the comment for the cluster
+		// setting admission.l0_sub_level_count_overload_threshold.
+		maxNumSubLevels := int64(math.Round(float64(iot.L0Size) / float64(iot.L0MinimumSizePerSubLevel)))
+		// Say numSubLevels is 30 and maxNumSubLevels is 5. We don't want to
+		// ignore the huge disparity, since that could allow the numSubLevels to
+		// grow to a huge value. So we place a lower-bound. The divisor of 3 was
+		// chosen somewhat arbitrarily.
```

Comment:
Perhaps add a sentence here about how the max can actually be lower than the min but the min takes precedence.